packages/web/src/pages/explore-page/components/desktop/NewExplorePage.tsx

```diff
@@ -119,13 +118,13 @@ const ExplorePage = ({ title, pageTitle, description }: ExplorePageProps) => {
   const navigate = useNavigate()
   const showSearchResults = useShowSearchResults()
 
-  const { data: featuredPlaylists } = useFeaturedPlaylists(
-    { limit: FEATURED_LIMIT },
-    { placeholderData: (prev: TQCollection[]) => prev }
-  )
-  const { data: featuredProfiles } = useFeaturedProfiles({
-    limit: FEATURED_LIMIT
-  })
+  const { data: exploreContent } = useExploreContent()
```

Comment:
so do we want all the data consolidated to one hook? i kinda think we should keep them separate so they can load asap? fine with this for now though!

Reply:
i think this way is actually slightly more performant because useFeaturedPlaylists for example will use useExploreContent and useCollections, getting more data when all we need is ids here